src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_task_commands.py

```diff
@@ -8,6 +8,7 @@
 
 class AcrTaskCommandsTests(ScenarioTest):
 
+    # @unittest.skip("task.py line 250, BUG: Discriminator type is absent or null, use base class TaskStepProperties.")
```

Comment:
is this comment still necessary?

Reply:
It's a bug we found in SDK, but there is still no response from codegen team.
We have made temporary fix in cli.

Reply:
so are we meant to skip or not skip this test?

Reply:
Actually, we don't want to skip this test. It's a reminder to fix the bug later.
Now just using a temporary solution